base/doc/ltnews39.tex

```diff
@@ -725,6 +725,19 @@ \subsection{Struts at the end of footnotes or \texttt{p} columns}
 \pkg{bigfoot})}
 
 
+\subsection{Fix a ``missing \cs{item}'' rollback error}
+
+If \LaTeX{} is rolled back to a date between 2023/06/01 (inclusive) and
+2024/06/01 (exclusive), any list-based environment would raise an error
```

Comment:
Maybe nowadays YYYY-MM-DD form is more preferable?

Reply:
hmm, I personally prefer the old one, but I don't mind either way

Reply:
Then let it be.